pkg/pillar/cmd/msrv/handlers.go

```diff
@@ -0,0 +1,719 @@
+// Copyright (c) 2024 Zededa, Inc.
```

Comment:
This is derived from zedrouter/metadata_handlers.go hence it should add 2024 to the existing copyright years in that file.

Reply:
Fixed